<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormatSettingsDialog</class>
 <widget class="QDialog" name="FormatSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Format settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Unrecognized char:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="unrecognizedChar">
       <item>
        <property name="text">
         <string>~</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">^</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">*</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">?</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">&amp;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>$</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>@</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="preserveLineBreaks">
     <property name="text">
      <string>Preserve line breaks</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Font formatting</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="useBold">
        <property name="text">
         <string>Use bold</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="useItalic">
        <property name="text">
         <string>Use italic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="useUnderlined">
        <property name="text">
         <string>Use underlined</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="useFontSize">
        <property name="text">
         <string>Use font size</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>useFontSize</zorder>
     <zorder>useBold</zorder>
     <zorder>useItalic</zorder>
     <zorder>useUnderlined</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Alternatives</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="showAlternatives">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Show alternatives as tooltip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="alternativesUnderline">
        <property name="text">
         <string>Underline characters with alternatives</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Alternative color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ColorButton" name="alternativeColorButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QToolButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FormatSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FormatSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
